src/frontend/App/typer/personopplysninger.ts

```diff
@@ -22,13 +22,19 @@ export interface IPersonopplysninger {
     oppholdstillatelse: IOppholdstillatelse[];
     vergemål: IVergemål[];
 }
-
+export interface IDeltBostedPeriode {
+    startdatoForKontrakt: string;
+    sluttdatoForKontrakt: string;
+}
 export interface IBarn {
     navn: string;
     personIdent: string;
     annenForelder?: IAnnenForelderMinimum;
     adresse: IAdresse[];
+
     borHosSøker: boolean;
```

Comment:
Med tanke på at logikken for borHosSøker er endret, burde også vilkårsinformasjonen på aleneomsorg oppdateres?

Reply:
Vilkår = neste PR

Reply:
Burde då denne og neste PR merges før backend? Då backend endrer på hvordan `borHosBruker` vises?
Dvs hvis vi merger backend nå så kan det vises nei, men så burde man få få tilleggsinformasjonen om at det er delt bosted.

Reply:
Ser nå at dere har lagt tilbake `if (harDelt` i backend